new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -733,21 +960,38 @@ public static boolean deletePvc(String name, String namespace)
     if (!response.isSuccess()) {
       logger.warning(
           "Failed to delete persistent volume claim '" + name + "' from namespace: "
-              + namespace + " with HTTP status code: " + response.getHttpStatusCode());
+          + namespace + " with HTTP status code: " + response.getHttpStatusCode());
       return false;
     }
 
     if (response.getObject() != null) {
       logger.info(
           "Received after-deletion status of the requested object, will be deleting "
-              + "persistent volume claim in background!");
+          + "persistent volume claim in background!");
     }
 
     return true;
   }
 
-  // --------------------------- service account ---------------------------
+  /**
+   * List all persistent volumes in the Kubernetes cluster.
+   * @return V1PersistentVolumeList of Persistent Volumes in Kubernetes cluster
+   */
+  public static V1PersistentVolumeList listPersistenVolumes() {
+    KubernetesApiResponse<V1PersistentVolumeList> list = pvClient.list();
+    return list.getObject();
+  }
 
+  /**
+   * List all persistent volume claims in the Kubernetes cluster.
```

Comment:
does this handle all namespaces?

Reply:
Yes it lists persistent volume claims in all namespaces. I also added another overloaded method to handle namespace specific list.

Reply:
Let's call it listPersistentVolumeClaimsAllNamespaces() then.  Also, why would we ever need this as the diagnostics should be per-namespace.

Reply:
changed the name to listPersistentVolumeClaimsAllNamespaces. I added this method when we need to query all namespaces in the cluster in case needed.

Reply:
where do we need this?  as opposed to just looking in a single namespace?

Reply:
removed listPersistentVolumeClaimsAllNamespaces()